Base.rte/Craft/Shared/MGTurret.lua

```diff
@@ -0,0 +1,104 @@
+function Create(self)
+	self.turnSpeed = 0.015;	-- Speed of the turret turning, in rad per frame
+	self.searchRange = 250;	-- Detection area diameter or max ray distance to search enemies from, in px
+	-- Whether turret searches for enemies in a larger area. Default mode is a narrower ray detection
+	self.areaMode = true;
+	-- Toggle visibility of the aim area / trace to see how the detection works
+	self.showAim = false;
+	-- Angle alteration variables (do not touch)
+	self.rotNum = 0;	-- Left / Right
+	self.vertiNum = 0;	-- Up / Down
```

Comment:
It's hard to explain what they are since they're just factors that track how the controller states affect the angle of the turret. Hopefully the new names make more sense.

Reply:
Yeah, naming is hard. I've known many a programmer (and been one myself) who's spent hours and even close to full workdays back-and-forthing on naming. No one's perfect at it, but it's worth working hard on all the same.